uiowa/uiowa
# Updated linkit to D10.0 compatible version

Code changes: https://git.drupalcode.org/project/linkit/-/compare/6.0.0-rc1...6.0.2

# How to test
```
ddev blt ds --site sandbox.uiowa.edu && ddev drush @sandbox.local uli node/add/page
```
- Test suggestions for content
- Test suggestions for webform
- Test paths that don't get suggestions (views page paths or paths that don't exist)
- Test external links
- Test anchor links
- Ensure that Title and Advanced properties still work

New functionality (unpublish highlight), resolves #6343 :

<img width="480" alt="Screenshot 2023-10-16 at 9 42 18 AM" src="https://github.com/uiowa/uiowa/assets/4663676/63c7aac6-4577-414b-a70b-8b3d11aa6e74">
